### ISSUE TITLE
[issue] Arabic support

### ISSUE THREAD

**Issue by dinarpay** [marc2332/solana-mobile-wallet#3]
Hello
Is the wallet fully open source? 
Can Arabic language be supported?
You can see an example of an Android app

Thanks

**Comment by biomassives** [comment]
Yes this wallet is 'fully open source.'
 MIT License

A short and simple permissive license with conditions only requiring preservation of copyright and license notices. Licensed works, modifications, and larger works may be distributed under different terms and without source code.
Permissions

Commercial use
Modification
Distribution

 Private use

Limitations

Liability

Warranty

Conditions

License and copyright notice

**Comment by marc2332** [comment]
Yes, it's completely open source. I plan to add other languages eventually 😄

**Comment by dinarpay** [comment]
Is it possible to see a trial version Android

**Comment by marc2332** [comment]
> Is it possible to see a trial version Android

The Wallet is still work in progress, it's not 100% secure, neither completely reliable. Still, you can build it from source code if you want
